src/models/darp10/external-overview.md

```diff
@@ -80,6 +80,7 @@
 |![Fn-F1](./img/fn-f1.png)  |Fn+F1   |Toggle trackpad                    |
 |![Fn-F2](./img/fn-f2.png)  |Fn+F2   |Mic mute                           |
 |![Fn-F3](./img/fn-f3.png)  |Fn+F3   |Mute                               |
+|![Fn-F4](./img/fn-star.png)  |Fn+F4   |Toggle keyboard backlight          |
```

Comment:
This should probably also have an asterisk and a footnote, since the 16" does not have a toggle hotkey on F4